pkg/secrets/handlers/secrets_handler_k8s.go

```diff
@@ -0,0 +1,130 @@
+package handlers
+
+import (
+	"fmt"
+	"github.com/cyberark/conjur-authn-k8s-client/pkg/access_token"
+	secretsConfig "github.com/cyberark/conjur-authn-k8s-client/pkg/secrets/config"
+	"github.com/cyberark/conjur-authn-k8s-client/pkg/secrets/conjur"
+	"github.com/cyberark/conjur-authn-k8s-client/pkg/secrets/k8s"
+	log "github.com/cyberark/conjur-authn-k8s-client/pkg/sidecar/logging"
+	"os"
+	"strings"
+)
+
+type SecretsHandlerK8sUseCase struct {
+	AccessTokenHandler   access_token.AccessTokenHandler
+	ConjurSecretsFetcher conjur.ConjurSecretsFetcherInterface
+	K8sSecretsHandler    k8s.K8sSecretsHandler
+}
+
+var errLogger = log.ErrorLogger
+
+func NewSecretHandlerK8sUseCase(secretsConfig secretsConfig.Config, AccessTokenHandler access_token.AccessTokenHandler) (SecretsHandler *SecretsHandlerK8sUseCase, err error) {
+	k8sSecretsHandler, err := k8s.New(secretsConfig)
+	if err != nil {
+		errLogger.Printf(err.Error())
+		os.Exit(1)
+	}
+
+	var conjurSecretsFetcher conjur.ConjurSecretsFetcher
+
+	return &SecretsHandlerK8sUseCase{
+		AccessTokenHandler:   AccessTokenHandler,
+		ConjurSecretsFetcher: conjurSecretsFetcher,
+		K8sSecretsHandler:    *k8sSecretsHandler,
+	}, nil
+}
+
+func (secretsHandlerK8sUseCase *SecretsHandlerK8sUseCase) HandleSecrets() error {
```

Comment:
change SecretsHandlerK8sUseCase(without *) it to copy by value since we are not changing anything in the struct